### DISCUSSION TITLE
How to speed up simulations in MPB?

### DISCUSSION THREAD

**Discussion by niubiworker** [NanoComp/meep#2927] [Q&A]
I have a 32-core CPU, which is convenient for running FDTD, but when I run MPB, less than 1/5 of the CPU cores are in use. Is there any way to improve this? I am using Python.
Thank you very much!

**Comment by stevengj** [comment]
https://mpb.readthedocs.io/en/latest/Scheme_User_Interface/#parallel-mpb

(Wrong repository)
